lib/fuse_lowlevel.c

```diff
@@ -1968,7 +1968,7 @@ static void do_lseek(fuse_req_t req, fuse_ino_t nodeid, const void *inarg)
 /* Prevent bogus data races (bogus since "init" is called before
  * multi-threading becomes relevant */
 static __attribute__((no_sanitize("thread")))
-void do_init(fuse_req_t req, fuse_ino_t nodeid, const void *inarg)
+void do_init(fuse_req_t req, fuse_ino_t nodeid, const void *inarg, bool buf_reallocable)
```

Comment:
I just don't see what's happening here with the code, it looks like you're passing a `size_t` to a `bool` parameter?

Reply:
Yeah, conversion from size to bool, I had seen it, but I would like to avoid that parameter altogether.

Reply:
Agreed, I'd like to see a different solution here but I'm surprised this compiled.